portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -14,7 +18,13 @@ public class HelloWorldServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<String> arrayList = new ArrayList<>();
```

Comment:
Always initialize as an interface. This should be:
List<String> arrayList = new ArrayList<>();